tappsbt/interface.go

```diff
@@ -529,6 +530,14 @@ type VOutput struct {
 	// serialized, this will be stored in the TaprootInternalKey and
 	// TaprootDerivationPath fields of the PSBT output.
 	ScriptKey asset.ScriptKey
+
+	// ProofSuffix is the optional new transition proof blob that is created
+	// once the asset output was successfully committed to the anchor
+	// transaction referenced above. The proof suffix is not yet complete
+	// since the header information needs to be added once the anchor
+	// transaction was confirmed in a block. This field cannot be of type
+	// proof.Proof directly because that would cause a circular dependency.
+	ProofSuffix []byte
```

Comment:
after reading the rest of the commits I think it's a bit unclear why this proof suffix needs to be exported now

previously we were only anchoring one vPSBT on a anchoringTx, why didn't that require this field too?

Reply:
See my long-form comment above, just before your review. Before this change the proofs never "left" the daemon, they were copied from one state to the next in a separate data structure within the freighter. But now, because we return the incomplete state back to the user, we need to transport the proofs somewhere. I think the vPSBT itself isn't too bad a place, but we might need to discuss that.

Reply:
Yeah I think this is a part of the more interactive vPSBT ceremony. When you're operating mainly outside the daemon, you want to prove to someone after signing that the new outputs was properly constructed (no duplicate commitments, etc), so you need to transmit this data _somehow_, and it makes sense to just piggy back it on top of what you were already sending to each other.